### PR TITLE
Web-mobile Guide

### PR DESCRIPTION
**Pull request by samipak458**
I would like to add my project as web-mobile guide.

A simple library management project that is easy to use and easy to understand. I created this website while learning web development. I used all the basic concepts of JS in this project and I want to share my project to help other learner to understand the basic concepts of JS. I made this project using HTML, Bootstrap and JS. I also hosted this website using Azure Static Web Apps service.

Website: [Online Library](https://delightful-coast-00b51a700.1.azurestaticapps.net/)

### CODE REVIEW

File: league-content/web-mobile.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ Resources of interest to students and faculty all about web and mobile developme
 [A simple clone of the game Wordle using React and tailwind CSS](https://github.com/MahmoudFettal/wordle)
 > This sample was submitted by Student Ambassador Mahmoud Fettal
 
+[A Simple Library Mangement using HTML, Bootstrap and JS](https://github.com/samipak458/Simple-Library-Managment)
```
Comment (jlooper) [inline league-content/web-mobile.md:11]: just a small typo: Management - looks good other than that!

**Review by jlooper — CHANGES_REQUESTED** [review]
just a small typo

**Comment by jlooper** [comment]
@samipak458 could you check whether this repo exists publicly and let's try again, sorry I didn't check and pre-emptively merged

**Comment by samipak458** [comment]
Yes, this repo exist publicly. No issues, I will create another pull request.

**Pull request merged**